backstop_data/engine_scripts/puppet/price-set/add-price-field-modal.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+const Page = require('../page-objects/crm-page.js');
+
+module.exports = async (engine, scenario, viewport) => {
+  const page = new Page(engine, scenario, viewport);
+  
+  await page.checkIfPriceFieldsAreEmpty('Fields')
+    .then(async () => {
```

Comment:
why do you need `then`? We're using `await` here (applies for all the other scripts)